python/packages/autogen-core/src/autogen_core/components/tools/_base.py

```diff
@@ -3,6 +3,7 @@
 from collections.abc import Sequence
 from typing import Any, Dict, Generic, Mapping, Protocol, Type, TypedDict, TypeVar, runtime_checkable
```

Comment:
```suggestion
from typing import Any, Dict, Generic, Mapping, Protocol, Type, TypedDict, TypeVar, cast, runtime_checkable
```

Reply:
Done in c67fd0b33e38f3d534beb9cd16f31facb09826d2